.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -0,0 +1,89 @@
+name: Release checklist
+description: Checklist for each release. This template is only for the core team.
+title: "[Release] Element Android v "
+labels: [\U0001F680 Release]
+assignees:
+  - bmarty
+
+body:
+  - type: textarea
+    id: checklist
+    attributes:
+      label: Release checklist
+      description: For the template example, we are releasing the version 1.1.10. Replace 1.1.10 with the version in the issue body.
```

Comment:
Is it possible to automatize this replacement using the GH issue template?
input: version, ex: "1.2.1" and then we reuse this value to generate the issue description and even for the issue title